doc/metrics.md

```diff
@@ -0,0 +1,286 @@
+# Metrics
+
+Both the Java client and the Clojure client have [Dropwizard
+Metrics](http://metrics.dropwizard.io/3.1.0/) support - they both accept as an
+option a `MetricRegistry` to which they will register http metrics for each
+request, as well as metrics for an metric-ids specified in the request options.
```

Comment:
Somewhere in this paragraph you might want to call out that this is an experimental feature?